iac-extensions/spring-config/build.gradle.kts

```diff
@@ -11,6 +11,9 @@ dependencies {
 
     api(project(":iac-common"))
 
+    compileOnly(project(":iac-extensions:cloudformation"))
+    compileOnly(project(":iac-extensions:kubernetes"))
```

Comment:
Tbh I'm not that big of a fan of having this dependency, but there's no other way except duplicating and moving it to common.